packages/menu/src/Menu/Menu.types.ts

```diff
@@ -67,4 +67,11 @@ export interface MenuProps extends Omit<PopoverProps, 'active'> {
    * id passed to the menu dropdown.
    */
   id?: string;
+
+  /**
+   * Whether the menu should always render dark, regardless of the theme context
+   *
+   * @default {true}
+   */
+  renderDarkMenu?: boolean;
```

Comment:
When `renderDarkMenu` is true

<img width="294" alt="Screenshot 2024-06-26 at 1 22 35 PM" src="https://github.com/mongodb/leafygreen-ui/assets/5740159/1677104b-92b1-451e-942f-9ae732af6354">

Reply:
Since `MenuGroup` didn't exist before, and we're deprecating the dark menus soon, I didn't spend time getting this prop combo to work

Reply:
also noticed that the collapse/expand icon button coloring for the submenu doesn't have enough contrast
<img width="509" alt="Screenshot 2024-06-26 at 12 01 30 PM" src="https://github.com/mongodb/leafygreen-ui/assets/52300529/6dd11e6a-fb16-4842-8315-34c8fd3129bd">